sdk/storage/test-resources-post.ps1

```diff
@@ -46,44 +53,44 @@ $content =
+      <TableServiceSecondaryEndpoint>https://$PremiumAccountName-secondary.$PremiumAccountZone.table.$EndpointSuffix</TableServiceSecondaryEndpoint>
+      <ConnectionString>DefaultEndpointsProtocol=https;AccountName=$PremiumAccountName;AccountKey=$PremiumAccountKey;BlobEndpoint=https://$PremiumAccountName.$PremiumAccountZone.blob.$EndpointSuffix;FileEndpoint=https://$PremiumAccountName.$PremiumAccountZone.file.$EndpointSuffix;QueueEndpoint=https://$PremiumAccountName.$PremiumAccountZone.queue.$EndpointSuffix;TableEndpoint=https://$PremiumAccountName.$PremiumAccountZone.table.$EndpointSuffix</ConnectionString>
     </TenantConfiguration>
     <TenantConfiguration>
       <TenantName>ProductionTenant2</TenantName>
       <TenantType>Cloud</TenantType>
       <AccountName>$SecondaryAccountName</AccountName>
       <AccountKey>$SecondaryAccountKey</AccountKey>
-      <BlobServiceEndpoint>https://$SecondaryAccountName.blob.core.windows.net</BlobServiceEndpoint>
-      <QueueServiceEndpoint>https://$SecondaryAccountName.queue.core.windows.net</QueueServiceEndpoint>
-      <TableServiceEndpoint>https://$SecondaryAccountName.table.core.windows.net</TableServiceEndpoint>
-      <FileServiceEndpoint>https://$SecondaryAccountName.file.core.windows.net</FileServiceEndpoint>
-      <BlobServiceSecondaryEndpoint>https://$SecondaryAccountName-secondary.blob.core.windows.net</BlobServiceSecondaryEndpoint>
-      <QueueServiceSecondaryEndpoint>https://$SecondaryAccountName-secondary.queue.core.windows.net</QueueServiceSecondaryEndpoint>
-      <FileServiceSecondaryEndpoint>https://$SecondaryAccountName-secondary.file.core.windows.net</FileServiceSecondaryEndpoint>
-      <TableServiceSecondaryEndpoint>https://$SecondaryAccountName-secondary.table.core.windows.net</TableServiceSecondaryEndpoint>
+      <BlobServiceEndpoint>https://$SecondaryAccountName.$SecondaryAccountZone.blob.$EndpointSuffix</BlobServiceEndpoint>
+      <QueueServiceEndpoint>https://$SecondaryAccountName.$SecondaryAccountZone.queue.$EndpointSuffix</QueueServiceEndpoint>
+      <TableServiceEndpoint>https://$SecondaryAccountName.$SecondaryAccountZone.table.$EndpointSuffix</TableServiceEndpoint>
+      <FileServiceEndpoint>https://$SecondaryAccountName.$SecondaryAccountZone.file.$EndpointSuffix</FileServiceEndpoint>
+      <BlobServiceSecondaryEndpoint>https://$SecondaryAccountName-secondary.$SecondaryAccountZone.$EndpointSuffix</BlobServiceSecondaryEndpoint>
+      <QueueServiceSecondaryEndpoint>https://$SecondaryAccountName-secondary.$SecondaryAccountZone.$EndpointSuffix</QueueServiceSecondaryEndpoint>
+      <FileServiceSecondaryEndpoint>https://$SecondaryAccountName-secondary.$SecondaryAccountZone.$EndpointSuffix</FileServiceSecondaryEndpoint>
+      <TableServiceSecondaryEndpoint>https://$SecondaryAccountName-secondary.$SecondaryAccountZone.$EndpointSuffix<TableServiceSecondaryEndpoint>

```

Comment:
this looks different than other entires. i.e. theres no blob/queue/table inside.

Reply:
good catch. done.